frontend/src/app/testQueue/AoEForm/AoEForm.scss

```diff
@@ -0,0 +1,19 @@
+div.usa-form-group--error {
+  .usa-form-group--error {
+    border-left-width: 0;
+    padding-left: 0;
+
+    @media screen and (min-width: 1024px) {
+      border-left-width: 0.25rem;
+      padding-left: 1rem;
```

Comment:
Yes, and also using unit tokens for spacing (margin, padding, widths, etc) could be nice too, e.g. `padding-left: units(2);` is how I'd typically do this in our main SCSS file where I have access to token functions.